lib/iris/fileformats/rules.py

```diff
@@ -897,13 +897,19 @@ def _make_cube(field, converter):
     # Convert the field to a Cube.
     metadata = converter(field)
 
-    cube = iris.cube.Cube(field.core_data(),
+    cube_data = field.core_data()
+    cube_dtype = cube_data.dtype
+    if cube_dtype.kind in 'bui':
```

Comment:
Right, everywhere else this is expressed as `'biu'`... could you perhaps maintain that consistency?